docs/wiki/[User-Guide]-Starter-Module-Terraform-Sovereign-Landing-Zone.md

```diff
@@ -0,0 +1,111 @@
+<!-- markdownlint-disable first-line-h1 -->
+The `sovereign_landing_zone` starter module provides full customization of the Sovereign Landing Zone (SLZ) using the `inputs.yaml` file. The `inputs.yaml` file provides the ability to enable and disable modules, configure module inputs and outputs, and configure module resources.
+A custom `inputs.yaml` file can be passed to the `inputs` argument of the ALZ PowerShell Module. This allows you to firstly design your Azure Landing Zone, and then deploy it.
+
+The default `inputs.yaml` file will need to be modified based on the documentation below.
+
+Example input files can be found here:
+
+- [inputs-azure-devops-terraform-sovereign-landing-zone.yaml][example_powershell_inputs_azure_devops_terraform_sovereign_landing_zone]
+- [inputs-github-terraform-sovereign-landing-zone.yaml][example_powershell_inputs_github_terraform_sovereign_landing_zone]
+- [inputs-local-terraform-sovereign-landing-zone.yaml][example_powershell_inputs_local_terraform_sovereign_landing_zone]
+
+The following table describes the inputs for the `sovereign_landing_zone` starter module.
+
+| Input | Required | Type | Default Value | Description |
```

Comment:
This looks like the comprehensive list of inputs. Originally, this had just the required ones, but should this be all of them? Some of the optional inputs have descriptions that call out to other sections that were only included in the accelerator doc. So, if we should have the full list here, we should also pull over the other sections.